kotlin/create-symbol-styles-from-web-styles/README.md

```diff
@@ -0,0 +1,46 @@
+# Create symbol styles from web styles
+
+Create symbol styles from a style file hosted on a portal.
+
+![Image of create symbol styles from web styles](create-symbol-styles-from-web-styles.png)
```

Comment:
it looks like the image has been vertically stretched?

Reply:
Yeah :/. Currently all screenshots on Android are 700x400 pixels (so they render ok on github and in the sample viewer). It's a bit of a legacy from when phones used to be be shorter than the are these days. For now we'll just leave them as is and review all screenshots at a later date.